### PR TITLE
bundle: Separate bundles for user and and admin

### LINKED ISSUE #563: PanEuropean: Admin and Users bundles
- [ ] Create a separate bundle for both of them 
- [ ] Treat those pages without Navigation and without selected country (same as home)

### PR DESCRIPTION
**Pull request by sorja**
Resolves #563

### CODE REVIEW

File: webapp/app/actions.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import axios from 'axios'
 
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/actions.js:3]: it should be 
```
import { UserState } from '@webapp/store/user'
```

File: webapp/app/appViewExport.js [hunk 2]
```diff
@@ -35,10 +33,9 @@ const reducers = [
   { name: ReviewState.stateKey, fn: review },
   { name: UserManagementState.stateKey, fn: userManagement },
   { name: LandingState.stateKey, fn: landing },
-  { name: 'panEuropeanIndicators', fn: panEuropeanIndicators },
   { name: 'userChat', fn: userChat },
   { name: 'countryMessageBoard', fn: countryMessageBoard },
-  { name: AdminState.stateKey, fn: admin },
+  { name: AdminState.stateKey, fn: AdminReducer },
```
Comment (minotogna) [inline webapp/app/appViewExport.js:38]: do we still need admin here?

File: webapp/app/assessment/assessmentState.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { assessmentStatus } from '@common/assessment'
 
 import * as AppState from '@webapp/app/appState'
 import * as CountryState from '@webapp/app/country/countryState'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/assessment/assessmentState.js:12]: as above

File: webapp/app/assessment/fra/fraState.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { isPrintingMode } from '@webapp/pages/AssessmentPrint/printAssessment'
 import * as Assessment from '@common/assessment/assessment'
 
 import * as AppState from '@webapp/app/appState'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/assessment/fra/fraState.js:8]: as above

File: webapp/app/assessment/fra/sections/originalDataPoint/originalDataListView.js [hunk 5]
```diff
@@ -7,7 +7,7 @@ import { Link } from 'react-router-dom'
 import Icon from '@webapp/components/icon'
 
 import * as AppState from '@webapp/app/appState'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/assessment/fra/sections/originalDataPoint/originalDataListView.js:10]: as above

File: webapp/app/components/audit/actions.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import axios from 'axios'
 
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/components/audit/actions.js:3]: as above

File: webapp/app/countryLanding/views/linksView.js [hunk 7]
```diff
@@ -9,7 +9,7 @@ import { isStatusSaving } from '@webapp/app/components/autosave/autosave'
 import Icon from '@webapp/components/icon'
 import useI18n from '@webapp/components/hooks/useI18n'
 
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/countryLanding/views/linksView.js:12]: as above

File: webapp/app/countryLanding/views/manageCollaboratorsView.js [hunk 8]
```diff
@@ -21,7 +21,7 @@ import {
 } from '@webapp/app/user/userManagement/actions'
 
 import * as AppState from '@webapp/app/appState'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/countryLanding/views/manageCollaboratorsView.js:24]: as above

File: webapp/app/countryLanding/views/messageBoard/messageBoardPanel.js [hunk 9]
```diff
@@ -10,7 +10,7 @@ import { getRelativeDate } from '@webapp/utils/relativeDate'
 import { profilePictureUri } from '@common/userUtils'
 
 import * as AppState from '@webapp/app/appState'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/countryLanding/views/messageBoard/messageBoardPanel.js:13]: as above

File: webapp/app/countryLanding/views/messageBoard/users.js [hunk 10]
```diff
@@ -6,7 +6,7 @@ import { useDispatch, useSelector } from 'react-redux'
 import * as LandingState from '@webapp/app/countryLanding/landingState'
 import { i18nUserRole, profilePictureUri } from '@common/userUtils'
 import Icon from '@webapp/components/icon'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/countryLanding/views/messageBoard/users.js:9]: as above

File: webapp/app/user/userManagement/edit/editUserForm.js [hunk 11]
```diff
@@ -19,7 +19,7 @@ import TextInput from '@webapp/components/textInput'
 
 import * as AppState from '@webapp/app/appState'
 import * as CountryState from '@webapp/app/country/countryState'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/user/userManagement/edit/editUserForm.js:22]: as above

File: webapp/app/user/userManagement/editUserView.js [hunk 12]
```diff
@@ -7,7 +7,7 @@ import EditUserForm from './edit/editUserForm'
 
 import { isAdministrator } from '@common/countryRole'
 
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/app/user/userManagement/editUserView.js:10]: as above

File: webapp/components/Header/UserLinks/LinksFRA/LinksFRA.js [hunk 13]
```diff
@@ -9,7 +9,7 @@ import { useI18n, useIsLogin, useUserInfo } from '@webapp/components/hooks'
 import { PopoverControl } from '@webapp/components/popoverControl'
 import Icon from '@webapp/components/icon'
 
-import { logout } from '@webapp/user/actions'
+import { logout } from '@webapp/store/user/actions'
```
Comment (minotogna) [inline webapp/components/Header/UserLinks/LinksFRA/LinksFRA.js:12]: this should be 
```
import { UserActions } from '@webapp/store/user'
```

File: webapp/components/hooks/useUserInfo.js [hunk 14]
```diff
@@ -1,4 +1,4 @@
 import { useSelector } from 'react-redux'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/components/hooks/useUserInfo.js:2]: as above

File: webapp/main/rootReducer.js [hunk 15]
```diff
@@ -1,12 +1,12 @@
 import applicationError from '@webapp/components/error/reducer' // Reducers available before login
-import user from '@webapp/user/reducer'
+import user from '@webapp/store/user/reducer'
```
Comment (minotogna) [inline webapp/main/rootReducer.js:2]: as above

File: webapp/main/rootReducer.js [hunk 16]
```diff
@@ -1,12 +1,12 @@
...
 import app from '@webapp/app/reducer'
 import login from '@webapp/pages/Login/reducer'
 import country from '@webapp/app/country/reducer'
 
 import * as AppState from '@webapp/app/appState'
 import * as ApplicationErrorState from '@webapp/components/error/applicationErrorState'
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/main/rootReducer.js:9]: as above

File: webapp/pages/Admin/Admin.js [hunk 17]
```diff
@@ -0,0 +1,54 @@
...
+import VersioningView from './components/versioning/versioningView'
+
+import Menu from './components/Menu'
+
+const Admin = (props) => {
+  const userInfo = useSelector(UserState.getUserInfo)
+  const i18n = useI18n()
+  const { path, url } = useRouteMatch()
+
+  // Todo : redirect to /404 or /notfound
```
Comment (minotogna) [inline webapp/pages/Admin/Admin.js:24]: remove todo

File: webapp/pages/Admin/components/UserManagementView/index.js [hunk 18]
```diff
@@ -0,0 +1 @@
+export { default } from './usersManagementView'
```
Comment (minotogna) [inline webapp/pages/Admin/components/UserManagementView/index.js:1]: did you format with prettier?

File: webapp/pages/Admin/components/versioning/versioningView.js [hunk 19]
```diff
@@ -3,14 +3,14 @@ import './versioningViewStyle.less'
 import React, { useEffect } from 'react'
 import { connect } from 'react-redux'
 import { Switch, Route, useRouteMatch } from 'react-router-dom'
-import { getVersions, createVersion, deleteVersion, onChangeNewVersionForm } from '../../actions'
+import { getVersions, createVersion, deleteVersion, onChangeNewVersionForm } from '../../../../store/admin/actions'
```
Comment (minotogna) [inline webapp/pages/Admin/components/versioning/versioningView.js:6]: import AdminActions from index

File: webapp/pages/Assessment/Assessment.js [hunk 20]
```diff
@@ -17,8 +17,8 @@ import { useInitCountry, useIsCountryStatusLoaded } from '@webapp/store/country'
 import AssessmentHome from '@webapp/pages/Assessment/AssessmentHome'
 import AssessmentSectionView from '@webapp/app/assessment/components/section/assessmentSectionView'
 import OriginalDataPointView from '@webapp/app/assessment/fra/sections/originalDataPoint/originalDataPointView'
-import AdminView from '@webapp/pages/Admin/adminView'
-import EditUserView from '@webapp/app/user/userManagement/editUserView'
+import AdminView from '@webapp/pages/Admin/Admin'
+import EditUserView from '@webapp/pages/User/User'
```
Comment (minotogna) [inline webapp/pages/Assessment/Assessment.js:21]: Admin and User pages should be removed from here

File: webapp/store/user/reducer.js [hunk 21]
```diff
@@ -1,9 +1,9 @@
 import { exportReducer } from '@webapp/utils/reduxUtils'
 
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/userState'
```
Comment (minotogna) [inline webapp/store/user/reducer.js:3]: as above

File: webapp/store/user/reducer.js [hunk 22]
```diff
@@ -1,9 +1,9 @@
...
 
 import { appI18nUpdate, appInitDone } from '@webapp/app/actions'
-import { appUserLogout } from '@webapp/user/actions'
+import { appUserLogout } from '@webapp/store/user/actions'
```
Comment (minotogna) [inline webapp/store/user/reducer.js:6]: as above

**Review by minotogna — CHANGES_REQUESTED** [review]

File: webapp/app/actions.js [hunk 23]
```diff
@@ -1,6 +1,6 @@
 import axios from 'axios'
 
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/state'
```
Comment (minotogna) [inline webapp/app/actions.js:3]: ```import { UserState } from '@webapp/store/user'```

File: webapp/pages/User/export.js [hunk 24]
```diff
@@ -0,0 +1,14 @@
+import { UserState } from '@webapp/store/user'
+import user from '@webapp/store/user/reducer'
```
Comment (minotogna) [inline webapp/pages/User/export.js:2]: mport { UserReducer} from '@webapp/store/user'

File: webapp/store/user/reducer.js [hunk 25]
```diff
@@ -1,9 +1,9 @@
 import { exportReducer } from '@webapp/utils/reduxUtils'
 
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/state'
```
Comment (minotogna) [inline webapp/store/user/reducer.js:3]: ```import { UserState } from '@webapp/store/user'```

**Review by minotogna — CHANGES_REQUESTED** [review]
also, check unaddressed previous comments

File: webapp/app/actions.js (hunk 23 above)
Comment (sorja) [reply webapp/app/actions.js:3]: This gives cycle error, see commit [1bdb0fa](https://github.com/openforis/fra-platform/pull/606/commits/1bdb0fa5f2572b7ee004ffb6dacc898334cf160b)

File: webapp/store/user/reducer.js (hunk 25 above)
Comment (sorja) [reply webapp/store/user/reducer.js:3]: This gives cycle error, see commit [1bdb0fa](https://github.com/openforis/fra-platform/pull/606/commits/1bdb0fa5f2572b7ee004ffb6dacc898334cf160b)

**Review by minotogna — APPROVED** [review]

**Pull request merged**